packages/components/nodes/chatmodels/ChatGoogleGenerativeAI/ChatGoogleGenerativeAI.ts

```diff
@@ -74,6 +76,72 @@ class GoogleGenerativeAI_ChatModels implements INode {
                 step: 0.1,
                 optional: true,
                 additionalParams: true
+            },
+            {
+                label: 'topK',
+                name: 'topK',
+                type: 'number',
+                step: 0.1,
```

Comment:
step should be 1. 

**from the documentation**
_Top-K changes how the model selects tokens for output. A top-K of 1 means the next selected token is the most probable among all tokens in the model's vocabulary (also called greedy decoding), while a top-K of 3 means that the next token is selected from among the three most probable tokens by using temperature.
For each token selection step, the top-K tokens with the highest probabilities are sampled. Then tokens are further filtered based on top-P with the final token selected using temperature sampling.

Specify a lower value for less random responses and a higher value for more random responses.


Range: 1-40

Default for gemini-pro-vision: 32

Default for gemini-pro: none_